src/Microsoft.DotNet.ProjectJsonMigration/MSBuildExtensions.cs

```diff
@@ -15,6 +15,79 @@ namespace Microsoft.DotNet.ProjectJsonMigration
 {
     public static class MSBuildExtensions
     {
+        public static bool IsEquivalentTo(this ProjectItemElement item, ProjectItemElement otherItem)
+        {
+            // Different includes
+            if (item.IntersectIncludes(otherItem).Count() != item.Includes().Count())
+            {
+                MigrationTrace.Instance.WriteLine("ms: includes");
```

Comment:
I'm not sure, but these 3 "ms: foo" trace statements seem to be too short to be of value when looking through a log.  What does "ms" mean/stand for?